lib/ansible/modules/packaging/os/flatpak.py

```diff
@@ -125,6 +125,7 @@
 '''
 
 import subprocess
+import codecs
```

Comment:
You don't need codecs.. use ansible.module_utils.to_bytes

Reply:
Thank you for the tip! Having these in `module_utils` is indeed very convenient.

I applied your suggested changes.